db/seeds.rb

```diff
@@ -68,6 +68,18 @@
   o.admin = true
 end
 
+owner = Owner.find_or_create_by!(user_name: "rob") do |o|
+  o.real_name = "Rob Pritchard"
+  o.website = nil
+  o.email = "rob_pritchard@outlook.com"
+  o.password = "password" # placeholder password for seed data
+  o.country = "GB" # United Kingdom
+  o.real_name_visibility = :public
+  o.country_visibility = :public
```

Comment:
The first user already has public country visibility, it might be worth making this private or members only to make testing that feature easier!